java/src/main/kotlin/org/jetbrains/research/testspark/java/JavaPsiHelper.kt

```diff
@@ -138,37 +142,48 @@ class JavaPsiHelper(private val psiFile: PsiFile) : PsiHelper {
     }
 
     override fun getInterestingPsiClassesWithQualifiedNames(
-        cut: PsiClassWrapper,
+        cut: PsiClassWrapper?,
         psiMethod: PsiMethodWrapper,
     ): MutableSet<PsiClassWrapper> {
-        val interestingPsiClasses = cut.getInterestingPsiClassesWithQualifiedNames(psiMethod)
+        // The cut is always not null for Java, because all functions are always inside the class
+        val interestingPsiClasses = cut!!.getInterestingPsiClassesWithQualifiedNames(psiMethod)
         log.info("There are ${interestingPsiClasses.size} interesting psi classes from method ${psiMethod.methodDescriptor}")
         return interestingPsiClasses
     }
 
-    override fun getCurrentListOfCodeTypes(e: AnActionEvent): Array<*>? {
-        val result: ArrayList<String> = arrayListOf()
+    override fun getCurrentListOfCodeTypes(e: AnActionEvent): List<Pair<CodeType, String>> {
```

Comment:
It is better to have an alias for this `Pair<CodeType, String>`, as I cannot understand what exactly is returned.